src/main/java/uk/gov/companieshouse/api/accounts/validation/CreditorsWithinOneYearValidator.java

```diff
@@ -80,11 +67,8 @@ private Errors validateIfEmptyResource(CreditorsWithinOneYear creditorsWithinOne
         return errors;
     }
 
-    public Errors validateCreditorsWithinOneYear(@Valid CreditorsWithinOneYear creditorsWithinOneYear,
```

Comment:
Is there now an unused import for the `@Valid` annotation now?

Reply:
Resolved, removed the tag. Thanks brad!